jest/ImagePopout.spec.js

```diff
@@ -0,0 +1,50 @@
+import { mount } from '@vue/test-utils'
+import ImagePopout from '@/components/ImagePopout'
+
+describe('ImagePopout', () => {
+  test('should fill slot in container and modal with image provided', () => {
+    const wrapper = mount(ImagePopout, {
+      slots: {
+        default: `<img>`
+      }
+    })
+    expect(wrapper.html()).toMatchSnapshot()
+  })
+
+  test('should display caption if provided', () => {
+    const wrapper = mount(ImagePopout, {
+      propsData: {
+        caption: "Image Caption"
+      },
+      slots: {
+        default: `<img>`
+      }
+    })
+    expect(wrapper.html()).toMatchSnapshot()
+  })
+
+  test('should open modal when image is clicked', () => {
+    const wrapper = mount(ImagePopout, {
+      slots: {
+        default: `<img>`
+      }
+    })
+
+    wrapper.find('.syn-image-shadowed').trigger('click')
+
+    expect(wrapper.html()).toMatchSnapshot()
+  })
+
+  test('should close modal when modal image is clicked', () => {
+    const wrapper = mount(ImagePopout, {
+      slots: {
+        default: `<img>`
+      }
+    })
+
+    wrapper.find('.syn-image-shadowed').trigger('click')
```

Comment:
Using classes for selectors here feels a bit brittle since the tests will break if we ever change this class. Let's use an approach similar to the e2e tests. `wrapper.find('[jest="image]')`.

Reply:
Our jest tests will break anyhow if we change class names, though.